packages/ui-components/Icons/InstallationMethod/N.tsx

```diff
@@ -0,0 +1,34 @@
+import type { FC, SVGProps } from 'react';
+
+const N: FC<SVGProps<SVGSVGElement>> = props => (
+  <svg
+    width="32"
+    height="32"
+    viewBox="0 0 32 32"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    // terminal background (with border for on black background)
```

Comment:
Can we remove these inline comments? Not sure these are understood by the spec as code comments

Reply:
Will do.